### PR TITLE
mac80211:  add support for RTL8812au/RTL8821au USB Wi-Fi adapters.

### PR DESCRIPTION
**Pull request by mj22226**
This adds support for RTL8812au/RTL8821au USB wireless adapters.
Tested adapters: Alfa AWUS036ACS and  D-Link DWA-171 A1.

### CODE REVIEW

**Comment by Neustradamus** [comment]
@mj22226: Good job for this new supports?

File: package/kernel/mac80211/patches/rtl/045-v6.13-wifi-rtw88-Enable-the-new-RTL8821AU-RTL8812AU-driver.patch [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+--- a/drivers/net/wireless/realtek/rtw88/Kconfig
++++ b/drivers/net/wireless/realtek/rtw88/Kconfig
+@@ -13,8 +13,8 @@ if RTW88
+ 
+ config RTW88_CORE
+ 	tristate
+-	select WANT_DEV_COREDUMP
+ 	depends on m
++	select BPAUTO_WANT_DEV_COREDUMP
+
```
Comment (hauke) [inline package/kernel/mac80211/patches/rtl/045-v6.13-wifi-rtw88-Enable-the-new-RTL8821AU-RTL8812AU-driver.patch:85]: I think this change is not needed.

File: package/kernel/mac80211/patches/rtl/005-v6.12-wifi-rtw88-select-WANT_DEV_COREDUMP.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ drivers/net/wireless/realtek/rtw88/Kconfig | 1 +
+ 1 file changed, 1 insertion(+)
+
+--- a/drivers/net/wireless/realtek/rtw88/Kconfig
++++ b/drivers/net/wireless/realtek/rtw88/Kconfig
+@@ -13,6 +13,7 @@ if RTW88
+ 
+ config RTW88_CORE
+ 	tristate
++	select WANT_DEV_COREDUMP
```
Comment (hauke) [inline package/kernel/mac80211/patches/rtl/005-v6.12-wifi-rtw88-select-WANT_DEV_COREDUMP.patch:22]: Please select `BPAUTO_WANT_DEV_COREDUMP` here.

**Comment by Neustradamus** [comment]
Thanks to @mj22226 and @hauke! :)

**Pull request merged**